inspire_records/api/base.py

```diff
@@ -31,8 +31,45 @@
 from inspire_schemas.api import validate as schema_validate
 from invenio_pidstore.errors import PIDDoesNotExistError
 from invenio_records_files.api import Record
+from invenio_records.models import RecordMetadata
 from invenio_db import db
 
+from sqlalchemy import Text, or_, not_, cast, type_coerce
+from sqlalchemy.dialects.postgresql import JSONB
+
+class InspireQueryBuilderError(Exception):
+    pass
+
+class InspireQueryBuilder:
```

Comment:
if this is supposed to be Python 2 compatible, you want to inherit from `object`, otherwise you’ll get an old-style class.